cmd/cluster-controller/main.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2021 The KCP Authors.
+Copyright 2021-2022 The KCP Authors.
```

Comment:
we don't do this. Just leave 2021, i.e. when the files was created.

Reply:
Sorry, that was my wishy-washy advice in another PR. @RAMESSESII2 please undo this change & sorry for the confusion!

Reply:
sure,  no worries